Error while checking build/O2Physics/code-check for 3be5c2efd51cf624ad04f8b3e4233355eeafbe24 at 2026-08-18 10:50:
```
## sw/BUILD/O2Physics-code-check-latest/log
--
========== List of issues found ==========
++ echo 'Found 2 errors and 0 warnings.'
Found 2 errors and 0 warnings.
++ [[ 2 -gt 0 ]]
++ cat /sw/BUILD/541f06184d072c9ed31687c7f2c74fe52db490bd/O2Physics-code-check/errors.txt
PWGLF/Tasks/Strangeness/lambdaTwoPartPolarization.cxx:543:37: error: repeated branch body in conditional chain [bugprone-branch-clone,-warnings-as-errors]
PWGLF/Tasks/Strangeness/lambdaTwoPartPolarization.cxx:575:37: error: repeated branch body in conditional chain [bugprone-branch-clone,-warnings-as-errors]
++ [[ 0 -gt 0 ]]
++ [[ 2 -gt 0 ]]
++ exit 1
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/17491/3be5c2efd51cf624ad04f8b3e4233355eeafbe24/build_O2Physics_code-check/pretty.html).

--- a/PWGLF/Tasks/Strangeness/lambdaTwoPartPolarization.cxx
+++ b/PWGLF/Tasks/Strangeness/lambdaTwoPartPolarization.cxx
@@ -574,7 +574,7 @@ struct LfLambdaTwoPartPolarization {
       centrality = collision.centFT0C();
     } else if (cfgCentEst == kFT0M) {
       centrality = collision.centFT0M();
-    } 
+    }
     if (!eventSelected(collision) && cfgEvtSel) {
       return;
     }

--- a/PWGLF/Tasks/Strangeness/lambdaTwoPartPolarization.cxx
+++ b/PWGLF/Tasks/Strangeness/lambdaTwoPartPolarization.cxx
@@ -538,11 +538,10 @@ struct LfLambdaTwoPartPolarization {
                              TrackCandidates const& tracks, aod::V0Datas const& V0s,
                              aod::BCsWithTimestamps const&)
   {
+    centrality = collision.centFT0M();
     if (cfgCentEst == kFT0C) {
       centrality = collision.centFT0C();
     } else if (cfgCentEst == kFT0M) {
-      centrality = collision.centFT0M();
-    } else {
       centrality = collision.centFT0M();
     }
     if (!eventSelected(collision) && cfgEvtSel) {
@@ -570,13 +569,12 @@ struct LfLambdaTwoPartPolarization {
                        TrackCandidates const& /*tracks*/, aod::V0Datas const& V0s,
                        aod::BCsWithTimestamps const&)
   {
+    centrality = collision.centFT0M();
     if (cfgCentEst == kFT0C) {
       centrality = collision.centFT0C();
     } else if (cfgCentEst == kFT0M) {
       centrality = collision.centFT0M();
-    } else {
-      centrality = collision.centFT0M();
-    }
+    } 
     if (!eventSelected(collision) && cfgEvtSel) {
       return;
     }

